Packs/CiscoWebExFeed/Integrations/CiscoWebExFeed/CiscoWebExFeed.yml

```diff
@@ -0,0 +1,125 @@
+  defaultvalue: Good
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators
+    fetched from the feed
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  required: false
+  type: 15
+- defaultvalue: indicatorType
+  display: ""
+  name: feedExpirationPolicy
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- defaultvalue: "20160"
+  display: ""
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: "30"
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Supports CSV values.
+  defaultvalue: WebEx
+  display: Tags
+  name: feedTags
+  required: false
+  type: 0
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  defaultvalue: "true"
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+description: The WebEx IP Address and Domain web site provided by Cisco to document
+  IPs and Domains used by WebEx. The WebEx Feed integration fetches indicators from
+  the web page, with which you can create a list (whitelist, blacklist, EDL, etc.)
+  for your SIEM or firewall service to ingest and apply to its policy rules.
+display: Cisco WebEx Feed
+name: Cisco WebEx Feed
+script:
+  commands:
+  - arguments:
+    - defaultValue: "100"
+      description: The maximum number of results to return. The default value is 10.
+      name: limit
+    - auto: PREDEFINED
+      defaultValue: Both
+      description: The indicator type. Can be "IPs", "URLs", or "Both". The default
+        value is "Both".
+      name: indicator_type
+      predefined:
+      - IPs
+      - URLs
+      - Both
+    description: Gets indicators from the feed.
+    name: webex-get-indicators
+  dockerimage: demisto/btfl-soup:1.0.1.12087
+  feed: true
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
+fromversion: 5.0.0
```

Comment:
please change to fromversion: 6.0.0

Reply:
Yeah I didn't have an official source, so I reused the one that was in xsoar already.